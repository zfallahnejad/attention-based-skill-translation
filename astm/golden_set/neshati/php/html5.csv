1415724,html5
