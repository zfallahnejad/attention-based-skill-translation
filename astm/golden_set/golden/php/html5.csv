1415724,12,3,0.8
